pkg/job/update_user_roles.go

```diff
@@ -26,12 +26,12 @@ func (c *updateUserRoles) Run() error {
 	for _, guild := range guilds.Data {
 		err = c.updateLevelRoles(guild.ID)
 		if err != nil {
-			return err
+			c.log.Error(err, "updateLevelRoles failed")
```

Comment:
why not return error

Reply:
when return error, update nft role below cannot run -> not update nft role